src/processUserGuess.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const { WINNING_STATUS } = require('./constants');
+
+function processUserGuess(randomNumber, guess) {
+  if (guess === randomNumber) {
+    return WINNING_STATUS;
+  }
+
+  const stringRandomNumber = randomNumber.toString().split('');
+  const stringGuess = guess.toString().split('');
+
+  const isBull = stringRandomNumber
+    .some((numb, index) => +numb === +stringGuess[index]);
+
+  if (isBull) {
+    return 'bull';
+  }
+
+  const isCow = stringRandomNumber
+    .some(numb => stringGuess.includes(numb));
+
+  if (isCow) {
+    return 'cow';
+  }
```

Comment:
<img width="757" alt="image" src="https://github.com/mate-academy/node_bulls-and-cows/assets/86789161/e1d63a1e-a16f-43f6-8f8e-b200692222cb">

You should return number of bulls and cows, not just message
For example:
generated number: 1234
user entered number: 2135

You should print message with `bulls: 1, cows: 2`